joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/base/AstLinkerPass.scala

```diff
@@ -16,6 +16,9 @@ class AstLinkerPass(cpg: Cpg) extends CpgPass(cpg) with LinkingUtil {
     cpg.typeDecl.whereNot(_.astParent).foreach { typeDecl =>
       addAstParent(typeDecl, typeDecl.fullName, typeDecl.astParentType, typeDecl.astParentFullName, dstGraph)
     }
+    cpg.member.whereNot(_.astParent).foreach { member =>
+      addAstParent(member, member.typeFullName, member.astParentType, member.astParentFullName, dstGraph)
```

Comment:
`member.typeFullName` is not this member's full name. Members don't really have full names, though, but this is only used to log errors. Perhaps you could do something like 
```suggestion
      addAstParent(member, s"${member.astParentFullName}.${member.name}", member.astParentType, member.astParentFullName, dstGraph)
```